canonical-web-and-design/jaas-dashboard
# Add Jest snapshots to existing components

## Done

Add Jest snapshots to existing components 📸

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Run `./run test`
- Verify no errors
